bazel/canbench.sh

```diff
@@ -12,12 +12,10 @@
 
 set -eEuo pipefail
 
-# Whether to update the results file.
-UPDATE=${1:-}
-
 RUNFILES="$PWD"
 REPO_PATH="$(dirname "$(readlink "$WORKSPACE")")"
 REPO_RESULTS_PATH="${REPO_PATH}/${CANBENCH_RESULTS_PATH}"
+CANBENCH_OUTPUT=/tmp/canbench_output.txt
```

Comment:
Is this a correct way of storing the output? Won't benchmark tests interfere with each other?

Reply:
What about:
```suggestion
CANBENCH_OUTPUT="$(mktemp)"
```